website/server/models/user/schema.js

```diff
@@ -521,6 +521,7 @@ let schema = new Schema({
     return {};
   }},
   pushDevices: [PushDeviceSchema],
+  ABtest: {type: String},
```

Comment:
User is different than other models. Everything is noset by default. It should be made private though.